pydocusign/test.py

```diff
@@ -59,6 +59,7 @@ def docusign_client_factory(settings=os.environ):
         username=settings['PYDOCUSIGN_TEST_USERNAME'],
         password=settings['PYDOCUSIGN_TEST_PASSWORD'],
         integrator_key=settings['PYDOCUSIGN_TEST_INTEGRATOR_KEY'],
+        **kwargs
```

Comment:
Cannot pass "integrator_key" in kwargs. Explicit key/value in kwargs should have priority over (override) settings.
What about something like:

```python
options = {
    username=settings['PYDOCUSIGN_TEST_USERNAME'],
}
options.update(kwargs)
client = DocuSignClient(**kwargs)
```

?